shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/sql/supported/ddl/drop.xml

```diff
@@ -23,6 +23,7 @@
     <sql-case id="drop_table_restrict" value="DROP TABLE t_log RESTRICT" db-types="MySQL,PostgreSQL" />
     <sql-case id="drop_table_cascade" value="DROP TABLE t_log CASCADE" db-types="MySQL,PostgreSQL" />
     <sql-case id="drop_table_cascade_constraints_and_purge" value="DROP TABLE t_log CASCADE CONSTRAINTS PURGE" db-types="Oracle" />
+    <sql-case id="drop_table_purge" value="DROP TABLE t_log PURGE" db-types="Oracle" />
```

Comment:
I suppose `DROP TABLE from db PURGE` is a more valuable case. Could you add this one or replace that simple one?

Reply:
Okay, I'll do it @tristaZero

Reply:
`DROP TABLE db.name PURGE`: is it the query we need to specify there?

Reply:
Hi sorry for my mistake. You're right.